src/utils/args.test.ts

```diff
@@ -1,6 +1,48 @@
-import { parse, validate, Config } from './args';
+import { parse, validate, Config, getStackNameFromFileName } from './args';
 import fs from 'fs';
 
+describe('The getStackNameFromFileName function', () => {
+  test('strips file extension', () => {
+    expect(getStackNameFromFileName('test.ts')).toBe('Test');
+  });
+
+  test('strips multiple extensions', () => {
+    expect(getStackNameFromFileName('test.spec.ts')).toBe('Test');
+  });
+
+  test('converts snake case to pascal case', () => {
+    expect(getStackNameFromFileName('test_name')).toBe('TestName');
+  });
+
+  test('converts kebab case to pascal case', () => {
+    expect(getStackNameFromFileName('test-name')).toBe('TestName');
+  });
+
+  test('converts camel case to pascal case', () => {
+    expect(getStackNameFromFileName('testName')).toBe('TestName');
+  });
+
+  test('converts mixed case to pascal case', () => {
+    expect(getStackNameFromFileName('this-is_aTestName')).toBe(
+      'ThisIsATestName'
+    );
+  });
+
+  test('removes any other special characters', () => {
+    expect(
+      getStackNameFromFileName('t!e@s£t$-%n^a&m*e()_-+=[]{}|\\"\';:/?><,~`')
```

Comment:
🤣

Reply:
Figured I'd just try it with all the special chars I could find on my keyboard. Although just realised that I missed `§` and `±` 😱